Source/ALSV4_CPP/Public/Character/ALSBaseCharacter.h

```diff
@@ -422,6 +430,9 @@ class ALSV4_CPP_API AALSBaseCharacter : public ACharacter
 	UFUNCTION(Category = "ALS|Replication")
 	void OnRep_OverlayState(EALSOverlayState PrevOverlayState);
 
+	UFUNCTION(Category = "ALS|Replication")
+    void OnRep_VisibleMesh(USkeletalMesh* NewVisibleMesh);
```

Comment:
Indentation

Reply:
Good spot, must fix that in Rider (tabs vs spaces)